app/models/child_approval.rb

```diff
@@ -9,6 +9,7 @@ class ChildApproval < UuidApplicationRecord
   has_many :nebraska_approval_amounts, dependent: :destroy
   has_many :attendances, dependent: :destroy
   has_many :service_days, -> { distinct }, through: :attendances
+  has_one :notification, dependent: :destroy
```

Comment:
Out of curiosity, why are you also associating to a child_approval?  Was this a mistake?  I ask because I don't see you using this association anywhere.

Reply:
Yes this was a mistake.